src/components/forms/DateInput.jsx

```diff
@@ -240,6 +244,7 @@ const DateInput = ({
           IconRight={DatePickerIcon}
         />
       }
+      {...popoverProps}
```

Comment:
I can see both sides of why we would / wouldn't do this, but we sure we want to use rest props here?

Reply:
`Popover` itself does not spread props on any DOM elements, so I figured this was safe to do here.